### ISSUE TITLE
Capture Agent title/status at time of contribution?

### ISSUE THREAD

**Issue by mbrush** [data2health/contribution-data-model#24]
Should we include an attribute on the Contribution object that lets us capture the professional position/title/status of an agent when the contribution was made? This would support queries like "_Find papers with more than 4 contributions by graduate students"_

Or should this be handled as part of the Agent model which we defer to implementations to define?

**Comment by mellybelly** [comment]
I agree this is useful information but i worry it gets too much into modeling people. however it is a sort of secondary transitive attributution to the environment/context/org in which the contribution was made.

I guess i would err on keeping it and getting feedback. People do want to be able to query for these sorts of contributions all the time: what are the products of my post-docs? for example.

**Comment by mellybelly** [comment]
I might leave this out upon further thinking - it seems to be the type of modeling that would be in places like VIVO, i guess i don't think person status belongs in this model? obviously the questions are interesting but they should be answserable when you combine data from expertise/profiling systems? good point though about how to answer these questions when there is no other data or system to do the person association to these types of contexts.
